bosh-templates/cloud_controller_clock.yml.erb

```diff
@@ -76,6 +76,9 @@ billing_event_writing_enabled: <%= p("cc.billing_event_writing_enabled") %>
 
 default_app_memory: <%= p("cc.default_app_memory") %>
 default_app_disk_in_mb: <%= p("cc.default_app_disk_in_mb") %>
+<% if_p("cc.default_app_file_descriptors") do |file_descriptors|%>
```

Comment:
Hi @shashidharatd,

You should not need to guard against this property not being set. Cloud Controller should be robust to this configuration value being null. You should follow the same pattern as the properties above it, e.g. `default_app_memory`. Can you also make this change for `bosh-templates/cloud_controller_worker.yml.erb` and `bosh-templates/cloud_controller_api.yml.erb` as well.

Thanks,

@crhino and @jfmyers9